assets/app/views/browse/pod.html

```diff
@@ -86,6 +86,22 @@
               </tab>
               -->
 
+              <tab heading="Terminal" active="selectedTab.terminal"
+                select="renderOptions.preventContainerSockets = false"
```

Comment:
Yeah renderOptions is usually reserved for things that effect what renders
in the chrome around the main content area, like the label filter widget.
You should just set this on something like
$scope.terminal.preventContainerSockets

@liggit this is preventing the websocket from connecting until the first
time you open the tab, but then allows it to remain open as long as you
just switch between the tabs on the pod so commands can keep running. It
only kills the connection if you leave the pod page.
On Oct 9, 2015 10:22 AM, "Jordan Liggitt" <notifications@github.com> wrote:

> In assets/app/views/browse/pod.html
> <https://github.com/openshift/origin/pull/5012#discussion_r41634939>:
>
> > @@ -86,6 +86,22 @@
> >                </tab>
> >                -->
> >
> > +              <tab heading="Terminal" active="selectedTab.terminal"
> > +                select="renderOptions.preventContainerSockets = false"
>
> renderOptions seems like an odd place to put this... what is this doing?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/openshift/origin/pull/5012/files#r41634939>.
>